mypy/types.py

```diff
@@ -197,6 +287,33 @@ def __init__(self, typ: mypy.nodes.TypeInfo, args: List[Type],
     def accept(self, visitor: 'TypeVisitor[T]') -> T:
         return visitor.visit_instance(self)
 
+    type_ref = None  # type: str
+
+    def serialize(self) -> JsonDict:
+        data = {'.class': 'Instance',
+                }  # type: JsonDict
+        if self.type is not None:
+            data['type_ref'] = self.type.alt_fullname or self.type.fullname()
+        if self.args:
+            data['args'] = [arg.serialize() for arg in self.args]
+        if self.erased:
```

Comment:
We don't need to preserve this when serializing, as `erased` only has an effect within expressions, and they aren't serialized. We can default to false always. (Actually, `erased` is not actually used right now, but the attribute can still prove to be useful in the future, so I'd rather preserve it in the full AST.)